src/components/Advertisements/Advertisements.spec.tsx

```diff
@@ -461,24 +471,22 @@ describe('Testing Advertisement Component', () => {
     await wait();
 
     const date = await screen.findAllByTestId('Ad_end_date');
-    const dateString = date[0].innerHTML;
+    const dateString = date[1].innerHTML;
     const dateMatch = dateString.match(
       /\b(?:Sun|Mon|Tue|Wed|Thu|Fri|Sat)\s+(Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)\s+(\d{1,2})\s+(\d{4})\b/,
     );
     let dateObject = new Date();
-
     if (dateMatch) {
       const monthName = dateMatch[1];
       const day = parseInt(dateMatch[2], 10);
       const year = parseInt(dateMatch[3], 10);
-
       const monthIndex =
         'JanFebMarAprMayJunJulAugSepOctNovDec'.indexOf(monthName) / 3;
 
       dateObject = new Date(year, monthIndex, day);
     }
 
-    expect(dateObject.getTime()).toBeLessThan(new Date().getTime());
+    expect(dateObject.getTime()).toBeGreaterThan(new Date().getTime());
```

Comment:
Why was this logic changed?

Reply:
Apologies, I mistakenly changed the logic for the date comparison, but I have since fixed it.